dojo/tools/gosec/parser.py

```diff
@@ -17,32 +16,37 @@ def get_description_for_scan_types(self, scan_type):
     def get_findings(self, filename, test):
         tree = filename.read()
         try:
-            data = json.loads(str(tree, 'utf-8'))
-        except:
+            data = json.loads(str(tree, "utf-8"))
+        except BaseException:
```

Comment:
```suggestion
        except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50